lms/static/js/learner_dashboard/views/program_header_view.js

```diff
@@ -0,0 +1,56 @@
+;(function (define) {
+    'use strict';
+
+    define(['backbone',
+            'jquery',
+            'text!../../../templates/learner_dashboard/program_header_view.underscore',
+            'picturefill'
+           ],
+         function(Backbone, $, pageTpl, picturefill) {
+            return Backbone.View.extend({
+                breakpoints: {
+                    min: {
+                        'medium': '768px',
+                        'large': '1180px',
+                        'x-large': '1440px'
+                    }
+                },
+
+                el: '.js-program-header',
+
+                tpl: _.template(pageTpl),
+
+                initialize: function() {
+                    this.render();
+                },
+
+                render: function() {
+                    var data = $.extend(this.model.toJSON(), {
+                        breakpoints: this.breakpoints
+                    });
+
+                    if ( !!this.model.get('name') ) {
```

Comment:
Why do we need two "!!" here for this condition? In here isn't it being used as double negative?